lib/rubygems/gem_runner.rb

```diff
@@ -74,6 +82,12 @@ def do_configuration(args)
     Gem.use_paths Gem.configuration[:gemhome], Gem.configuration[:gempath]
     Gem::Command.extra_args = Gem.configuration[:gem]
   end
+
+  def warn_on_outdated
+    if defined?(OpenSSL::SSL) && (Gem.latest_rubygems_version > Gem.rubygems_version)
+      say "You are currently using gem #{Gem.rubygems_version}, however gem #{Gem.latest_rubygems_version} is availble.\nConsider upgrading using the command `gem update --system`\n"
```

Comment:
I suggest to use `RubyGems` instead of `gem`. `gem` is command name, but common name for whole package manager is `RubyGems`.

"You are currently using RubyGems 3.0.4, however RubyGems 3.1.4 is available."

There's also currently typo in available (`availble`).